transcom/mymove
# INT-B-18638

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A904632&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

Added Wounded Warrior labels on move details pages, move queues, and customer pages

### How to test

Access the customer app
Login as a new user
Create a move with the Wounded Warrior order type
Test Wounded Warrior label on customer home page header and move details header
Login as a TOO/TIO/SC/QAE/Prime and search for new move
Make sure Warrior label is visible within the move queue and on the Move Details Header
